docs/contributing/contributing-code/contributing-code-tests/tests-images-pushtoghcr.md

```diff
@@ -0,0 +1,61 @@
+
+## Using Standard Images in Tests
+
+When writing tests for our applications, it's common to use standard images for resources like Redis, Mongo, and others. By default, specifying an image with `redis:latest` or similar tags will pull the image from Docker Hub. This behavior is due to Docker's default configuration, which automatically searches the Docker Hub registry for images when no other registry is specified. 
+
+However, to avoid potential rate limiting and network issues associated with pulling images from Docker Hub, we are transitioning to using images hosted in our public GitHub Container Registry (GHCR) repository. This change ensures more reliable access to the images needed for our tests and helps reduce potential disruptions caused by external dependencies.
+
+### New Image Pull Guideline
```

Comment:
I think we should just write this the way it is supposed to work now.....stay away from comparing old way to new.

Reply:
updated